### ISSUE TITLE
access feature request

### ISSUE THREAD

**Issue by kimpham54** [dulibrarytech/digitaldu-backend#217]
we have published/unpublished, another access option would be to show that we have a record in our system but do not provide direct access

[10:35 AM] Katherine Crowe
    So I have a legal and philosophical question that needs some technical weigh-in. I had Nathalie convert a video that's. student-made called "Bring Back Boone" from a DVD so I have the digital file and it has an item in ASpace. The video's backing track is Pon de Replay by Rihanna which (in addition to it making me angry that the students decided to put Rihanna as a backing track to this particular video) means sharing it would be a massive copyright violation. Still, it has historical significance and needs preservation. Is it possible to develop a process so we can preserve digital content that we can't share publicly? 
Edited​[10:36 AM] Katherine Crowe
    I think a secondary question is how can we share the metadata about any items like this so people KNOW we have it even if they can't see it so they can request to view it.
​[10:36 AM] Katherine Crowe
    No need to discuss right now but since it's come up I figured I'd ask the question and we can "parking lot" it until it makes sense to talk in more detail
(1 liked)
[11:03 AM] Kim Pham
    so is it just ingesting metadata
​[11:03 AM] Kim Pham
    where do you keepthe videos?

[11:04 AM] Katherine Crowe
    I think it would be fine to just get sucked into Archivematica the problem is we have no functional way to provide streaming access to these videos.
​[11:04 AM] Katherine Crowe
    The Jerome Robbins Dance Division has some of this built into their repository sort of
​[11:05 AM] Katherine Crowe
    https://digitalcollections.nypl.org/collections/jerome-robbins-archive-of-the-recorded-moving-image-original-documentation#/?tab=navigation
​[11:05 AM] Katherine Crowe
    what they use though I have no idea
​[11:08 AM] Katherine Crowe
    Basically they make patrons come into the reading room so it is entirely possible that we don't need to provide restricted streaming access
​[11:09 AM] Katherine Crowe
    How the heck that works right now I have no idea
​[11:12 AM] Kim Pham
    i think we have a lot of the functionality in place already
​[11:12 AM] Kim Pham
    we do have published/unpublished, maybe we can do a third access option
​[11:12 AM] Kim Pham
    i will add it to the list
(1 liked)

**Comment by freyesdulib** [comment]
Add "is_publish=2" option 
Add path to thumbnail containing "This item may be viewed onsite..." OR include message in the metadata (no access to asset)
Index metadata into public index

**Comment by kcrowe1981** [comment]
I looked at this with Nathalie and I think we need to be clearer on what a specific example would look like (on our/SCA's end). Is this proposed solution in your comment @freyesdulib  assuming that there is no Kaltura link at all? I think that would make the most sense, and the next step, which is "what does a virtual reading room look like" if we DID want to try to provide remote access on a case by case basis, would come after. We have done this a few times, essentially uploaded a file to Kaltura, made a private link, shared it for a short duration of time, and then deleted it - that is probably the most "quick and dirty" solution, but it does require a lot of manual back and forth, which doesn't seem very scalable.
